src/page-multi-facility/ScenarioSidebar.tsx

```diff
@@ -1,11 +1,18 @@
 import PromoBoxWithButton from "../design-system/PromoBoxWithButton";
 import ToggleRow from "./ToggleRow";
+import { saveScenario } from "../database";
+
+interface ScenarioSidebar = {
+  scenario: Object;
```

Comment:
Feel free to add a proper `Scenario` type in this file:  https://github.com/Recidiviz/covid19-dashboard/blob/master/src/page-multi-facility/types.tsx

Reply:
Will look something like this:

```javascript
export type Scenario = {
  name: string;
  baseline: boolean;
  dataSharing?: boolean;
  dailyReports?: boolean;
  roles: Map<String, String>;
  facilities?: Facilities;
};
```

Reply:
Will also include  `createdAt` and  `updatedAt` as required fields.  They're stored as `firebase.firestore.FieldValue.serverTimestamp();`, but I'm not sure what that translates to in the TypeScript/JavaScript Datetime world

Reply:
Okay I can add these!